evennia/utils/text2html.py

```diff
@@ -333,8 +362,16 @@ def format_styles(self, text):
                     # light grey text is the default, don't explicitly style
                     if color_class != "color-007":
                         classes.append(color_class)
+
                     # define the new style span
-                    prefix = '<span class="{}">'.format(" ".join(classes))
+                    if truecolor_fg == '' and truecolor_bg == '':
+                        prefix = f'<span class="{" ".join(classes)}">'
+                    else:
+                        # Classes can't be used for true color
+                        prefix = (f'<span '
+                                  f'class="{" ".join(classes)}" '
```

Comment:
You comment that classes cannot be used, yet the `class=` is set here? At the very least the comment is confusing.

Reply:
Sorry about that! Classes can't be used for colors because of the verbosity of truecolor BUT other classes can still be applied (for things like blink, etc). I will clarify the comment.